lib/improver/spotdata/extract_data.py

```diff
@@ -214,10 +214,10 @@ def make_cube(self, cube, data, sites):
         """
         time_coord = cube.coord('time')
         time_coord.convert_units('hours since 1970-01-01 00:00:00')
-        latitudes = [site['latitude'] for site in sites.itervalues()]
-        longitudes = [site['longitude'] for site in sites.itervalues()]
-        utc_offsets = [site['utc_offset'] for site in sites.itervalues()]
-        site_ids = sites.keys()
+        latitudes = [float(site['latitude']) for site in sites.itervalues()]
+        longitudes = [float(site['longitude']) for site in sites.itervalues()]
+        utc_offsets = [int(site['utc_offset']) for site in sites.itervalues()]
```

Comment:
Okay, it's now a float. I am still using it as an integer in the extrema routine, but that's not likely to be used anytime soon and it is noted as a shortcoming there.